code/game/objects/structures/crates_lockers/closets/secure/bar.dm

```diff
@@ -23,3 +23,22 @@
 	new /obj/item/reagent_containers/food/drinks/bottle/small/beer(src)
 	new /obj/item/reagent_containers/food/drinks/bottle/small/beer(src)
 	new /obj/item/reagent_containers/food/drinks/bottle/small/beer(src)
+
+/obj/structure/closet/secure_closet/cabinet/beer
+	name = "beer closet"
+	desc = "A cabinet stacked end-to-end with six packs of beer."
+	req_access = null
+	storage_capacity = 45
+
+/obj/structure/closet/secure_closet/cabinet/beer/fill()
+	new /obj/item/storage/box/fancy/yoke/beer(src)
+	new /obj/item/storage/box/fancy/yoke/beer(src)
+	new /obj/item/storage/box/fancy/yoke/ebisu(src)
+	new /obj/item/storage/box/fancy/yoke/shimauma(src)
+	new /obj/item/storage/box/fancy/yoke/moonlabor(src)
+	new /obj/item/storage/box/fancy/yoke/earthmover(src)
+	new /obj/item/storage/box/fancy/yoke/whistlingforest(src)
+	new /obj/item/storage/box/fancy/yoke/threetowns(src)
+
+/obj/structure/closet/secure_closet/cabinet/beer/horizon
+	req_access = list(ACCESS_BAR)
```

Comment:
Don't re-add this, rice beer already has been added to be on-tap.

All of these are just reskins of ale, beer and rice beer, with the sole exception of threetowns. Just leave a yoke of threetowns in the back.

Reply:
I did however remove one of the generic yokes, just for lack of clutter's sake.

Reply:
Don't reintroduce the closet, just put them in the existing booze closet if you want to keep the yokes